assets/templates/datasetLandingPage/filterable.tmpl

```diff
@@ -142,11 +143,11 @@
              </li>
              {{ if .DatasetLandingPage.LatestChanges}}
              <li>
-              <div id="id-changes" class="section__content--markdown margin-right--0 margin-left-sm--2">
-                <h2 class="font-weight-700 padding-top--0">What has changed in this edition</h2>
+              <div id="id-changes" class="section__content--markdown margin-right--0">
+                <h2 class="font-weight-700 padding-top--0 cmd-heading">What has changed in this edition</h2>
```

Comment:
For all cmd-heading classes: I don't think we should use service/product specific terms (e.g. "cmd") in our code unless we really need to. And we should be able to achieve this styling by using `padding-left-sm--2 padding-left-md--3` and `padding-left--0`

Reply:
Implemented